sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/CHANGELOG.md

```diff
@@ -1,19 +1,19 @@
 # Release History
 
-## 5.2.0-beta.1 (Unreleased)
+## 5.2.0 (2023-02-23)
 
 ### Features Added
 
 - Added the an overload for `IAsyncCollector<EventData>` allowing a partition key to be specified.  Because `IAsyncCollector<T>` is owned by the Functions runtime, this method could not be directly added.  Instead, this has been implemented as an extension method within the Event Hubs extension package.  Unfortunately, this knowingly makes the overload unable to be mocked.
 
-### Breaking Changes
+- Target-based scaling support has been added, allowing instances for Event Hubs-triggered Functions to more accurately calculate their scale needs, adjust more quickly as the number of events waiting to be processed changes. This will also reduce duplicate event processing as the instance count changes.
```

Comment:
```suggestion
- Target-based scaling support has been added, allowing instances for Event Hubs-triggered Functions to more accurately calculate their scale needs and adjust more quickly as the number of events waiting to be processed changes. This will also reduce duplicate event processing as the instance count changes.
```

Reply:
I'll commit this suggestion once the live test pipeline completes.